microsetta_interface/templates/survey.jinja2

```diff
@@ -5,11 +5,60 @@
     <script type="text/javascript" src="/static/vendor/js/vue-2.5.17.min.js"></script>
     <script type="text/javascript" src="/static/vendor/vue-form-generator-2.3.4/vfg.js"></script>
     <link rel="stylesheet" type="text/css" href="/static/vendor/vue-form-generator-2.3.4/vfg.css">
+    <style type="text/css">
+        .vue-form-generator .field-radios .radio-list label {
+            padding-right: 25px;
+            display: inline-block;
+        }
+
+        .survey-nav-button {
+            padding: 6px 12px;
+            color: #ffffff!important;
+            background-color: #337ab7!important;
+            border: 1px solid #cccccc;
+            border-radius: 4px;
+            font-size: 14px;
+            font-weight: 400;
+            line-height: 1.42857143;
+            text-align: center;
+            display: none;
+        }
+
+        .form-group label span{
+            font-weight: 700;
+        }
+
+        .form-group {
+            padding-bottom: 5px;
+        }
+
+        .field-wrap {
+            font-size: 90%;
+        }
+
+        #form-progress {
+            width: 100%;
+            text-align: center;
+            display: none;
+            font-weight: bold;
+        }
+    </style>
     <script>
         var result_txt = "";
         var error_txt = "";
+        var form_submitted = 0;
 
         function postSurvey() {
+            fieldsetObject = document.getElementsByTagName("fieldset");
```

Comment:
Could this be declared with either `let` or `var`? Same comment for the declarations in the subsequent functions